mars/assetlist.json

```diff
@@ -18,6 +18,16 @@
       "name": "Mars",
       "display": "mars",
       "symbol": "MARS",
+      "traces": [
+        {
+          "type": "legacy-mintage",
```

Comment:
Still not quite right because of the self-referencing pointer. It's in the right location now, but should be referencing neutron's MARS (to answer the question: this is a legacy mintage of what other token?)